sympy/functions/combinatorial/factorials.py

```diff
@@ -829,14 +829,13 @@ class binomial(CombinatorialFunction):
 
     .. math:: \binom{n}{k} = \frac{ff(n, k)}{k!}
 
-    The following identity can be used to handle negative or
-    non-integer values of ``k`` when ``n - k`` is a non-negative
-    integer::
+    The following identity can be used to handle negative integral
```

Comment:
You can compute `binomial(3/2, -5/2)` as `binomial(3/2, 4)` which is why I wrote it like I did. But how about
```suggestion
    The following identity can be used to handle negative
```